dev-client/components/projects/ProjectsSearchView.tsx

```diff
@@ -0,0 +1,55 @@
+import {Badge, Box, FlatList, HStack, Input, VStack} from 'native-base';
+import {ProjectPreview} from '../../types';
+import {useTranslation} from 'react-i18next';
+import AddButton from '../common/AddButton';
+import MaterialIcon from '../MaterialIcon';
+import ProjectPreviewCard from './ProjectPreviewCard';
+
+type Props = {
+  projects: ProjectPreview[];
+};
+
+export default function ProjectsSearchView({projects}: Props) {
+  const {t} = useTranslation();
+  return (
+    <VStack bg="grey.200" p={5} flexGrow={1} flexShrink={0} flexBasis="70%">
+      <Box alignItems="flex-start" pb={3}>
+        <AddButton text={t('projects.create_button')} />
+      </Box>
+      <HStack alignContent="center">
+        <VStack>
+          <Badge
+            alignSelf="flex-end"
+            mb={-5}
+            mr={-2}
+            py={0}
+            rounded="full"
+            zIndex={1}
+            bg="none">
+            {projects.length}
+          </Badge>
+          <MaterialIcon
+            name="filter-list"
+            iconButtonProps={{color: 'grey.200'}}
+            iconProps={{color: 'action.active', size: 'sm'}}
+          />
+        </VStack>
+        {/* TODO: translation function returns null, but placeholder only accepts
+        undefined */}
+        <Input
+          placeholder={t('search.placeholder') || undefined}
```

Comment:
follow-up comment: 18next seems to have support for making the `t` function aware of possible translation keys & template values? https://www.i18next.com/overview/typescript. could there also be a follow-up issue to look into this?